core/actionProxy/owplatform/knative.py

```diff
@@ -108,15 +108,15 @@ def preProcessHTTPContext(msg, valueData):
             tmpBody = msg.get('value', {})
             removeInitData(tmpBody)
             bodyStr = str(tmpBody)
-            valueData['__ow_body'] = base64.b64encode(bodyStr)
+            valueData['__ow_body'] = base64.b64encode(bodyStr.encode())
         valueData['__ow_query'] = flask.request.query_string
 
     namespace = ''
     if '__OW_NAMESPACE' in os.environ:
         namespace = os.getenv('__OW_NAMESPACE')
     valueData['__ow_user'] = namespace
     valueData['__ow_method'] = flask.request.method
-    valueData['__ow_headers'] = flask.request.headers
+    valueData['__ow_headers'] = { k: v for k, v in flask.request.headers.items() }
```

Comment:
This seems less like a typo and more like an array conversion issue... could you clarify?

Reply:
Typing as in type system. It was the last thing I tested and I forgot to commit before I pushed.